docs/usage.md

```diff
@@ -31,13 +31,21 @@ This setup script will remove and reinstall the project dependencies
 and rebuild the site's JavaScript and CSS assets.
 
 !!! note
-	You may also run `./backend.sh` or `./frontend.sh`
-	if you only want to re-build the backend or front-end, respectively.
+  You may also run `./backend.sh` or `./frontend.sh`
+  if you only want to re-build the backend or front-end, respectively.
+
+##### Setting environments
+
+The `NODE_ENV` environment variable can be set in your `.env` file to either
+`development` or `production`, which will affect how the build is made and what
+gulp tasks are available. To install dependencies of one environment
+or the other run `./frontend.sh` (dependencies and devDependencies)
+or `./frontend.sh production` (non-devDependencies).
```

Comment:
The relationship between `NODE_ENV` and the parameter passed to `frontend.sh` could be made a little more clear, here.

Reply:
Also, I think it might be more clear to say "dependencies only" instead of "non-devDependencies".

Reply:
Is that clear enough that it's talking about excluding https://github.com/cfpb/cfgov-refresh/blob/master/package.json#L67

Reply:
I guess to be totally clear, we should say "dependencies but not devDependencies".

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4196/commits/910da66a1b4234b1aca91fb2859915766ffa3672